### ISSUE TITLE
'desc' filter replaced by 'file-source'

### ISSUE THREAD

**Issue by RonaldEAM** [php-translation/symfony-bundle#211]
**Description:**
I use desc filter like so `<h1>{{ 'index.header0'|trans|desc('Welcome') }}</h1>`in twig and run `php bin/console translation:extract` command, it executes ok but what I see in XLIFF messages.en.xlf is something like:
``` XML
<unit id="EqJj6WI">
  <notes>
    <note category="file-source" priority="1">templates/default/index.html.twig:10</note>
    <note category="file-source" priority="1">templates/default/index.html.twig:10</note>
  </notes>
  <segment>
    <source>index.header0</source>
    <target></target>
  </segment>
</unit>
```

**Actual:**
desc filter is replaced by `<note category="file-source" priority="1">templates/default/index.html.twig:10</note>` in XLIFF file

**Expected:**
desc filter should be added properly as category in XLIFF file like `<note category="desc" priority="1">Welcome</note>`

Note: I'm using SF4

**Comment by mvrhov** [comment]
No the desc should be used as a default translation. Just like in JMSTranslation, but this is not yet supported

**Comment by Nyholm** [comment]
This is fixed by #218
